config/routes.rb

```diff
@@ -19,5 +19,9 @@
     end
   end
 
+  namespace :api do
+    resources :service_providers, only: [:index]
```

Comment:
I really like the explicit path style we have in IDP repo, WDYT about keeping it for readability?

```
get '/api/service_providers' => 'api/service_providers#index'
```